mbuild/tests/test_hoomdxml.py

```diff
@@ -0,0 +1,16 @@
+import mbuild as mb
+import numpy as np
+from mbuild.tests.base_test import BaseTest
+
+
+class TestHoomdXML(BaseTest):
+
+    def test_save(self, ethane):
+        ethane.save(filename='ethane.hoomdxml')
+
+    def test_save_forcefield(self, ethane):
+        ethane.save(filename='ethane-opls.hoomdxml',forcefield='opls')
```

Comment:
These tests won't run since foyer doesn't get installed on the CI.

You can add a decorator via pytest to skip it if foyer is not installed.